src/recipe/admin/src/module.ts

```diff
@@ -37,6 +38,7 @@ export default defineNuxtModule<ModuleOptions>({
     const resolver = createResolver(import.meta.url);
 
     // passing module's options to runtime config for further access
+     _nuxt.options.runtimeConfig.public.error = _options.app?.error;
```

Comment:
this is a bit weird. it's probably because error handling plugin should have an error page property. let's discuss